eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -117,6 +117,10 @@ stages:
                     ServiceName: ${{parameters.ServiceDirectory}}
                     ForRelease: true
 
+          - template: /eng/pipelines/templates/jobs/installation.check.yml
```

Comment:
We should make this steps in the verify release version job instead of being its own job.